catalog/security/security-app/src/main/resources/etc/org.codice.ddf.catalog.security.policy.metacard.MetacardAttributeSecurityPolicyPlugin.config

```diff
@@ -0,0 +1,11 @@
+intersectMetacardAttributes=[ \
+  ]
+service.pid="org.codice.ddf.catalog.security.policy.metacard.MetacardAttributeSecurityPolicyPlugin"
+unionMetacardAttributes=[ \
+  "security.classification\=classification", \
+  "security.codewords\=SCIcontrols", \
+  "security.dissemination-controls\=disseminationControls", \
+  "security.security.other-dissemination-controls\=nonICmarkings", \
+  "security.owner-producer\=releasableTo", \
```

Comment:
you are correct, I think the answer is I shouldn't be configuring this here, the mappings for attributes is utterly confusing but we don't have a default AuthzRealm policy config or any  claims configured for sts in alliance so this doesn't make sense.